packages/adapter-planetscale/src/planetscale.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/require-await */
 import type planetScale from '@planetscale/database'
+import { Debug } from '@prisma/debug'
```

Comment:
I see you decided to import `Debug` directly from `@prisma/debug`.
A benefit of the previous solution via `@prisma/driver-adapter-utils` was that `Debug` would also be available for any third party that develops a driver adapter and only has the utils as a dependency.

Worth importing via utils?